languages/scala/recursive_descent/Lexer_scala.mll

```diff
@@ -394,7 +394,6 @@ rule token = parse
 
         | "package"     -> Kpackage t
         | "import"   -> Kimport t
```

Comment:
maybe add a comment around here saying there are more keywords in Scala but they
are not lexed here because they are "soft keywords" which are valid only in Scala 3 ...

Reply:
Those soft keywords are recognized in Parser_scala_recursive_descent.ml, for example with
accept (ID_LOWER ("export", ab)) in_;